distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -807,7 +807,7 @@ type Vector a
     slice : Integer -> Integer -> Vector Any
     slice self start end = @Builtin_Method "Vector.slice"
 
-    ## ALIAS first, last, slice, sample
+    ## ALIAS first, last, sample, slice
```

Comment:
I'm curious what is the reasoning behind changing the ordering?

Reply:
the linter sorts alphabetically when checking.